developing/relaying/relayers.md

```diff
@@ -19,5 +19,6 @@
 | [Simply Staking](https://www.simply-vc.com.mt/) | info@simply-vc.com.mt <br> <img src="https://user-images.githubusercontent.com/95667791/163506878-e7e278b2-1cf6-4dac-bc13-e92f63600ee0.png" alt="Twitter" width="18"/> Twitter [@Simply_VC](https://twitter.com/Simply_VC) <br> <img src="https://user-images.githubusercontent.com/95667791/163506639-7d27e947-94c7-4740-bb46-8e5791e9fc6c.png" alt="Telegram" width="18"/> Telegram [Simply Staking chat](https://t.me/simplyvc) | :lock: Delegate to Simply Staking on: [Osmosis](https://wallet.keplr.app/#/osmosis/stake?modal=detail&chainId=osmosis-1&validator=osmovaloper1feh2keupglep6mvxf5c96eulh3puujjryj2h8v), Agoric, Akash, Bitsong, Celo, Cosmos Hub, e-Money, Desmos, Gravity Bridge, Ixo, Ki, Microtick, Oasis, Persistence, Stargaze, Sommolier, Dig, Evmos, Lum, OmniFlix <br> :pray: Donations accepted @ <sup><sub>`osmo1feh2keupglep6mvxf5c96eulh3puujjr79z5st`</sub></sup> | :heavy_dollar_sign: Offers Relaying as a Service (RaaS) (contact directly for pricing) <br> :information_desk_person: Supports new relayers | Cosmos Hub, Akash, OmniFlix |
+| [StakeLab](https://www.stakelab.fr/) | hello@stakelab.fr <br> <img src="https://user-images.githubusercontent.com/95667791/163506878-e7e278b2-1cf6-4dac-bc13-e92f63600ee0.png" alt="Twitter" width="18"/> Twitter [@StakeLab](https://twitter.com/StakeLab) <br> <img src="https://user-images.githubusercontent.com/95667791/163506639-7d27e947-94c7-4740-bb46-8e5791e9fc6c.png" alt="Telegram" width="18"/> Telegram [StakeLab chat](https://t.me/stakelab_fr) | :lock: Delegate to StakeLab on: [Osmosis](https://wallet.keplr.app/#/osmosis/stake?modal=detail&chainId=osmosis-1&validator=osmovaloper16q8xd335y38xk2ul67mjg27vdnrcnklt4wx6kt), Akash, Asset Mantle, Axelar, Band Protocol, Bitcanna, Bitsong, Cerberus, Certik, Chihuahua, Comdex, Cosmos Hub, Decentr, Desmos, Dig Chain, E-money, Evmos, Fetch Ai, Galaxy, Gravity Bridge, Injective, Iris, Ixo, Juno, Kava, Ki Chain, Konstellation, Lum, Medibloc, Meme, Microtick, Nomic, Omniflix, Osmosis, Persistence, Regen, Rizon, Secret, Sentinel, Sifchain, Sommelier, Stargaze, Starname, Terra, Umee, Vidulum | :information_desk_person: Supports new relayers | Almost all our blockchains supported |
```

Comment:
Can you please explicitly list out the chains relayed rather than saying 'almost all out blockchains...'?